### DISCUSSION TITLE
Get gyro/motion data from phone with freepie IMU to emulated dualshock 4

### DISCUSSION THREAD

**Discussion by ThePhaseless** [LizardByte/community#241] [Sunshine]
I know that adding native gyro support is almost impossible, but I think that adding a listener for [Freepie IMU](https://github.com/Xtreamer/FreePie-DiyHt/blob/master/Lib/Android/com.freepie.android.imu.apk) data and passing that into gyro of emulated dualshock 4 wouldn't be that hard and would greatly help to play fps games with devices like razer kishi

**Comment by ellitedev** [comment]
This would also greatly benefit Wii U emulation through CEMU. At that point, your phone becomes the Wii U GamePad.
